assets/styles/style.css

```diff
@@ -82,13 +89,18 @@ footer {
   justify-content: space-around;
   padding: 40px;
   height: 40px;
+  position: absolute;
```

Comment:
Unfortunately, using `position: absolute` here takes the footer out of the flow and causes it to overlap with the form when the viewport is small.

![Screenshot_2019-05-12 Mystery Code Society](https://user-images.githubusercontent.com/4643450/57587027-2f732c80-74c4-11e9-9e46-b6e720cd357b.png)

To work around this problem, you could a bottom margin to the content that is equal to the size of the footer, but that would be brittle. In my opinion, [flexbox-based approach](https://css-tricks.com/couple-takes-sticky-footer/#article-header-id-3) is the way to go.